.github/config/out_of_tree_extensions.cmake

```diff
@@ -86,8 +85,7 @@ if (NOT MINGW AND NOT ${WASM_ENABLED} AND NOT ${MUSL_ENABLED})
     duckdb_extension_load(iceberg
             ${LOAD_ICEBERG_TESTS}
             GIT_URL https://github.com/duckdb/duckdb-iceberg
-            GIT_TAG d62d91d8a089371c4d1862a88f2e62a97bc2af3a
-            APPLY_PATCHES
+            GIT_TAG 3060b30309d82f1059c928de7280286fcf800545
             )
 endif()
```

Comment:
Note: iceberg is currently wrapped in `if (NO) .. endif()`, that also needs removing (here or in a follow up)